x/concentrated-liquidity/swapstrategy/one_for_zero_test.go

```diff
@@ -95,8 +95,9 @@ func (suite *StrategyTestSuite) TestComputeSwapStepOutGivenIn_OneForZero() {
 			// sqrt_price_current + token_in / liquidity
 			expectedSqrtPriceNext:    sqrtPriceTargetNotReached,
 			expectedAmountInConsumed: defaultAmountOne.Sub(sdk.NewDec(100)).Ceil(),
-			expectedAmountOut:        amountZeroTargetNotReached,
-			expectedFeeChargeTotal:   sdk.ZeroDec(),
+			// subtracting 3 * smallest dec to account for truncations in favor of the pool.
+			expectedAmountOut:      amountZeroTargetNotReached.Sub(sdk.SmallestDec().MulInt64(3)),
```

Comment:
How was 3 selected here?

Reply:
I just reran the tests, noticed that this is by how much it differed, and accepted it since it's insignificant

I think what matters that the rounding direction is desired and the difference is small